cli/cmd/runtime/start.go

```diff
@@ -197,3 +199,13 @@ func StartCmd(cliCfg *config.Config) *cobra.Command {
 	}
 	return startCmd
 }
+
+func parseConnectorDefs(s string) []*rillv1.ConnectorDef {
```

Comment:
The `package runtime` should probably have its own type for this (maybe something like `runtime.Connector`, and call the field `Config` instead of `Defaults`.